lang/sk.json

```diff
@@ -145,141 +145,143 @@
-		"originalSize": "[MISSING]Original Size",
+		"title": "Zmeniť veľkosť",
+		"originalSize": "Pôvodná veľkosť",
 		"sizes": {
-			"custom": "[MISSING]Custom",
-			"large": "[MISSING]Large",
-			"max": "[MISSING]Max",
-			"medium": "[MISSING]Medium",
-			"small": "[MISSING]Small"
+			"custom": "Vlastné",
+			"large": "Veĺké",
+			"max": "Maximum",
+			"medium": "Stredné",
+			"small": "Malé"
 		}
 	},
 	"editImage": {
-		"adjust": "[MISSING]Adjust",
-		"apply": "[MISSING]Apply",
-		"confirmExit": "[MISSING]Are you sure you want to close? You have unsaved changes to the image.",
-		"crop": "[MISSING]Crop",
-		"downloadAction": "[MISSING]Downloading original image.",
-		"keepAspectRatio": "[MISSING]Keep Aspect Ratio",
-		"loading": "[MISSING]Loading image",
-		"presets": "[MISSING]Presets",
-		"reset": "[MISSING]Reset",
-		"resize": "[MISSING]Resize",
-		"rotate": "[MISSING]Rotate",
-		"rotateAntiClockwise": "[MISSING]90&deg; Left",
-		"rotateClockwise": "[MISSING]90&deg; Right",
-		"save": "[MISSING]Save",
-		"saveDialogFileExists": "[MISSING]File with the same name already exists in folder.",
-		"saveDialogOverwrite": "[MISSING]Overwrite File",
-		"saveDialogSaveAs": "[MISSING]Save as:",
-		"saveDialogTitle": "[MISSING]Save Changes",
-		"title": "[MISSING]Edit Image",
-		"transformationAction": "[MISSING]Applying transformations.",
-		"uploadAction": "[MISSING]Uploading edited image.",
+		"adjust": "Prispôsobiť",
+		"apply": "Použiť",
+		"confirmExit": "Naozaj chcete ukončiť program? Vašé zmeny nebudú uložené.",
+		"crop": "Orezať",
+		"downloadAction": "Sťahujem originálny súbor.",
+		"keepAspectRatio": "Udržať polomer",
+		"loading": "Nahrávam obrázok",
+		"presets": "Zmeniť",
+		"reset": "Zmazať",
+		"resize": "Orezať",
+		"rotate": "Otočiť",
+		"rotateAntiClockwise": "90&deg; Doľava",
+		"rotateClockwise": "[MISSING]90&deg; Doprava",
+		"save": "UložiŤ",
+		"saveDialogFileExists": "Súbor s rovnakým menom už existuje v tomto súbore.",
+		"saveDialogOverwrite": "Prepísať súbor",
+		"saveDialogSaveAs": "Uložiť ako:",
+		"saveDialogTitle": "Uložiť zmeny",
+		"title": "Upraviť obrázok",
+		"transformationAction": "Použiť zmeny.",
+		"uploadAction": "Nahrávam upravené súbory.",
 		"filters": {
-			"brightness": "[MISSING]Brightness",
-			"clip": "[MISSING]Clip",
-			"contrast": "[MISSING]Contrast",
-			"exposure": "[MISSING]Exposure",
-			"gamma": "[MISSING]Gamma",
-			"hue": "[MISSING]Hue",
-			"noise": "[MISSING]Noise",
-			"saturation": "[MISSING]Saturation",
-			"sepia": "[MISSING]Sepia",
-			"sharpen": "[MISSING]Sharpen",
-			"stackBlur": "[MISSING]Blur",
-			"vibrance": "[MISSING]Vibrance"
+			"brightness": "Jas",
+			"clip": "Strih",
+			"contrast": "Kontrast",
+			"exposure": "Expozícia",
+			"gamma": "Gamma",
+			"hue": "Odtieň",
+			"noise": "Sýtosť",
+			"saturation": "Saturícia",
+			"sepia": "Sépia",
+			"sharpen": "Ostrosť",
+			"stackBlur": "Zaostrenie",
+			"vibrance": "Citlivosť"
 		},
+		
+		//It doesn´t need to be translated.
```

Comment:
Please don't add additional comments directly into the file - just remove the `[MISSING]` part and it's clear. Thanks!